app/javascript/stylesheets/common.scss

```diff
@@ -0,0 +1,3 @@
+.notes-table-height {
+  height: calc(100vh - 160px);
+}
```

Comment:
Let's follow https://www.bigbinary.com/learn-rubyonrails-book/creating-reusable-components#7-in-1-sass-pattern

This should be within a `components` stylesheet folder with the name `_notes.scss`.